test/integration/KarmaTestRunner.it.ts

```diff
@@ -92,8 +104,12 @@ describe('KarmaTestRunner', function () {
         expectToHaveSuccessfulTests(runResult, 0);
         expectToHaveFailedTests(runResult, []);
         expect(runResult.status).to.be.eq(RunStatus.Error);
-        expect(runResult.errorMessages.length).to.equal(1);
-        expect(runResult.errorMessages[0].indexOf('ReferenceError: Can\'t find variable: someGlobalVariableThatIsNotDeclared\nat')).to.eq(0);
+        if (runResult.errorMessages) {

```

Comment:
Same as before, this shouldn't be needed. It makes tests harder to read. We shouldn't force null checks in tests, we want it to break anyway.